test/integration/default/controls/_mapdata_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Keep only first 2 digits from Ubuntu finger
+mapdata_file = "_mapdata/#{system.platform[:finger].split('.').first}.yaml"
+
+# Load the mapdata from profile https://docs.chef.io/inspec/profiles/#profile-files
+mapdata_dump = inspec.profile.file(mapdata_file)
+mapdata_dump = mapdata_dump.gsub("\n", "\r\n") if platform[:family] == 'windows'
```

Comment:
This is the method required for Windows when using text-based comparison rather that YAML-based, as mentioned here:

* https://github.com/saltstack-formulas/openssh-formula/pull/193#issuecomment-682430080

Reply:
> This is the method required for Windows when using text-based comparison rather that YAML-based, as mentioned here:

@dafyddj seems to [use something else](https://github.com/saltstack-formulas/openssh-formula/pull/193#issuecomment-709445531):

```ruby
mapdata_content = mapdata_file.content.encode(universal_newline: true)
```

Reply:
@baby-gnu Thanks for that.  I've adapted the use of `.content.encode(universal_newline: true)` in a slightly different way.  I'll highlight it in its own block, to make it easier to discuss.

Reply:
No longer relevant since using YAML-based comparison now.